README.md

```diff
@@ -90,32 +119,29 @@ working with this repository:
 
 ##### `flow-typed validate-defs`
 
-Verifies that all files under the `/definitions/` directory are structured 
-properly. **It does not run tests**, it only asserts that file and directory 
-names match the expected conventions.
+Verifies that all files under the `/definitions/` directory are structured and 
+named properly. **It does not run tests**, it only asserts that file and 
+directory names match the expected conventions.
+
+This command is run during CI.
 
 ##### `flow-typed run-tests [optional-pattern]`
 
-Runs *all* compatible versions of Flow over the each library definition with 
-it's tests to ensure that the tests pass as expected.
+For each libdef, find each test and run it with all compatible versions of Flow.
+If any errors arise that are not *`// $ExpectError`*, the test has failed.
 
 Note that this command assumes that the `/definitions/` directory is correctly 
-structured. You can always verify the structure with the 
-[`flow-typed validate-defs`](#flow-typed-validate-defs) command.
+structured. It should be run *after* running 
+[`flow-typed validate-defs`](#flow-typed-validate-defs).
 
 ##### `flow-typed update-cache [--debug]`
 
 By default flow-typed retrieves all available libdefs from its related upstream
-repository. To make this process more efficient, those libdefs will be cached to
-your local filesystem. Usually, the cache will automatically be updated after a
-certain grace period during a libdef installation, but sometimes it is useful to
+repository. To make this process more efficient, those libdefs arce cached once 
```

Comment:
Ok, so I updated this -- but GitHub makes it seem like I didn't...
(Not that we can merge this yet, just noting)